packages/dm-core-plugins/src/attribute-selector/Sidebar.tsx

```diff
@@ -18,19 +19,21 @@ export const Sidebar = (): JSX.Element => {
           onClick={() => setSelectedTab('home')}
           active={selectedTab === 'home'}
         />
-        {Object.values(childTabs).map((tabData: TChildTab) => (
-          <SideBar.Link
-            key={tabData.attribute}
-            icon={subdirectory_arrow_right}
-            label={
-              tabData.entity?.label ||
-              prettifyName(tabData.entity?.name || '') ||
-              tabData.attribute
-            }
-            onClick={() => setSelectedTab(tabData.attribute)}
-            active={selectedTab === tabData.attribute}
-          />
-        ))}
+        {Object.values(childTabs).map((tabData: TChildTab) => {
+          return (
```

Comment:
Could remove the extra curly brackets now

Reply:
fixed